### PR TITLE
Add NoAutoTupling test cases involving classes and case classes

### PR DESCRIPTION
**Pull request by gabro**
Just being paranoid here.

### CODE REVIEW

**Comment by gabro** [comment]
Hum, the build has failed for a timeout

**Comment by olafurpg** [comment]
Hmm, weird, I restarted the build.

cc/ @jvican the build timed out while syncing the cache, any idea what could have caused it? Any idea if we could setup auto-restart in these cases?

File: scalafix-tests/src/test/resources/checkSyntax/NoAutoTupling.source [hunk 1]
```diff
@@ -98,3 +98,25 @@ object tup {
...
+
+<<< auto-tupling with class constructors
+object tup {
+  class Foo(t: (Int, String))(s: (Boolean, List[Int]))
+  new Foo(1, "foo")(true, Nil)
+}
+>>>
+object tup {
+  class Foo(t: (Int, String))(s: (Boolean, List[Int]))
+  new Foo((1, "foo"))((true, Nil))
```
Comment (olafurpg) [inline scalafix-tests/src/test/resources/checkSyntax/NoAutoTupling.source]: Awesome! I'm impressed this works actually 😄

**Comment by olafurpg** [comment]
cc/ @jvican the build timed out while syncing the cache, any idea what could have caused it? Any idea if it's possible to setup auto-restart in these cases?

**Comment by jvican** [comment]
The drone documentation doesn't say anything about it, so I would bet it's not possible to auto restart it. I don't know what it may have caused it. Maybe flaky connection?

File: scalafix-tests/src/test/resources/checkSyntax/NoAutoTupling.source (hunk 1 above)
Comment (gabro) [inline scalafix-tests/src/test/resources/checkSyntax/NoAutoTupling.source]: So was I 😂

**Pull request merged**